src/libraries/System.Private.CoreLib/src/System/Text/Encoding.cs

```diff
@@ -75,6 +76,8 @@ namespace System.Text
 
     public abstract partial class Encoding : ICloneable
     {
+        private const string ENABLEUTF7_APPCONTEXT_SWITCH_NAME = "System.Text.Encoding.EnableUnsafeUTF7Encoding";
```

Comment:
>ENABLEUTF7_APPCONTEXT_SWITCH_NAME [](start = 29, length = 33)

Can we define this in the file [https://github.com/dotnet/runtime/blob/master/src/libraries/System.Private.CoreLib/src/System/LocalAppContextSwitches.cs](https://github.com/dotnet/runtime/blob/master/src/libraries/System.Private.CoreLib/src/System/LocalAppContextSwitches.cs)

just to be easy to find all config switches in one place